Presentation.Web/app/components/it-contract/it-contract-overview.controller.ts

```diff
@@ -976,6 +1021,12 @@
                         ],
                         uiState: [
                             "uiCustomizationStateService", (uiCustomizationStateService: Kitos.Services.UICustomization.IUICustomizationStateService) => uiCustomizationStateService.getCurrentState(Kitos.Models.UICustomization.CustomizableKitosModule.ItContract)
+                        ],
+                        criticalityOptions: [
+                            "localOptionServiceFactory",
+                            (localOptionServiceFactory: Services.LocalOptions.ILocalOptionServiceFactory) =>
+                            localOptionServiceFactory.create(Services.LocalOptions.LocalOptionType.CriticalityTypes)
```

Comment:
this endpoint only returns the currently available options which is fine for the selection box options, but since we use it for lookup it will fail (result in blank) on options which are registered and then expired by the local admin. (in the controller it is managed with the "existingOption". See in DPR's overview for "Overførselsgrundlag" on how to deal with this.

Reply:
Hint: It uses a custom endpoint which exports all needed types including their expiration status (uses generic services for it so it's easy to build)